src/EA.Iws.Web/Areas/AdminImportNotificationMovements/Controllers/CaptureController.cs

```diff
@@ -148,5 +152,15 @@ public ActionResult Cancelled(Guid id)
         {
             return View(id);
         }
+        private ImportInternalMovementSummary GetSummarydata(Guid id)
```

Comment:
As with the export, you can remove these methods and call the method directly on the view model.